src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java

```diff
@@ -76,10 +78,14 @@ public String key() {
 
   MediaHeartbeatConfig config;
   private MediaHeartbeat heartbeat;
+  HeartbeatFactory heartbeatFactory;
+  String heartbeatTrackingServer;
```

Comment:
Also for a style thing, these should also should be organized by their qualifier. Right now constants are being mixed in order with member variables.

Reply:
Just did this. Thanks!